cloudmarker/events/rdbmsenforcetlsevent.py

```diff
@@ -0,0 +1,100 @@
+"""RDBMS Enforce TLS/SSL Event.
+
+This module defines the :class:`RDBMSEnforceTLSEvent` class that
+identifies RDBMS servers which have TLS/SSL connection enforcement
+disabled. This plugin works on the  properties found in the ``com``
+bucket of ``rdbms`` records.
+"""
+
+
+import logging
+
+from cloudmarker import util
+
+_log = logging.getLogger(__name__)
+
+
+class RDBMSEnforceTLSEvent:
+    """Az RDBMS TLS/SSL enforcement event plugin."""
+
+    def __init__(self):
+        """Create an instance of :class:`RDBMSEnforceTLSEvent`."""
+
+    def eval(self, record):
+        """Evaluate RDBMS servers for TLS connection enforcement.
+
+        Arguments:
+            record (dict): A RDBMS record.
+
+        Yields:
+            dict: An event record representing an RDBMS where TLS
+            connection enforcement is disabled
```

Comment:
The second line "connection enforcement is disabled" needs to be indented one level more.

Reply:
From Google Python Style Guide section [doc function returns](https://google.github.io/styleguide/pyguide.html#doc-function-returns) no indentation is needed.
Thanks @susam  for the explanation.